### PR TITLE
chore: add IT test configuration

### PR DESCRIPTION
**Pull request by Neenu1995**
cc: @meltsufin @suztomo

### CODE REVIEW

**Review by meltsufin — APPROVED** [review]

File: bootstrap.sh [hunk 1]
```diff
@@ -19,6 +19,8 @@ git clone https://github.com/newren/git-filter-repo.git
 export PATH=$PATH:`pwd`/git-filter-repo
 
 mkdir google-cloud-java
+cp -r ../templates/.kokoro google-cloud-java
```
Comment (meltsufin) [inline bootstrap.sh:22]: Can we do it as a separate commit after line 45?

File: templates/.kokoro/build.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+set -eo pipefail
+
+## Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+## cd to the parent directory, i.e. the root of the git repo
+cd ${scriptDir}/..
+
+# include common functions
+source ${scriptDir}/common.sh
```
Comment (meltsufin) [inline templates/.kokoro/build.sh:24]: I don't think this file exists.

File: templates/.kokoro/build.sh [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+# Print out Maven & Java version
+mvn -version
+echo ${JOB_TYPE}
+
+mvn -pl -java-container clean install
```
Comment (meltsufin) [inline templates/.kokoro/build.sh:30]: Is this some kind of tests remnant?

File: bootstrap.sh (hunk 1 above)
Comment (Neenu1995) [reply bootstrap.sh]: fixed

File: templates/.kokoro/build.sh (hunk 2 above)
Comment (Neenu1995) [reply templates/.kokoro/build.sh:24]: Fixed

File: templates/.kokoro/build.sh (hunk 3 above)
Comment (Neenu1995) [reply templates/.kokoro/build.sh:30]: java container is a snowflake which uses JDST project for its IT tests. It will not work with gcloud-devel.
Until we figure out a way to switch project based on module or use a different project, java container is excluded from the IT tests.

**Review by meltsufin — APPROVED** [review]
